src/redux/actions/experiments/loadExperiments.js

```diff
@@ -0,0 +1,39 @@
+import fetchAPI from '../../../utils/fetchAPI';
+import {
+  EXPERIMENTS_LOADED,
+  EXPERIMENTS_ERROR,
+  EXPERIMENTS_LOADING,
+} from '../../actionTypes/experiments';
+
+const loadExperiments = (
+  projectUuid,
```

Comment:
I think we don't need to split. into three lines this part, right? or does the linter make a fuss about it too?

Reply:
No we don't and the linter isn't complaining, but I think it's easier to read